ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/StreamedQueryResource.java

```diff
@@ -198,12 +218,19 @@ private Response handleStatement(
         final PreparedStatement<Query> queryStmt = (PreparedStatement<Query>) statement;
 
         if (queryStmt.getStatement().isPullQuery()) {
-          return handlePullQuery(
+          final Response response =  handlePullQuery(
               securityContext.getServiceContext(),
               queryStmt,
               request.getConfigOverrides(),
               request.getRequestProperties()
           );
+          if (pullQueryExecutorMetrics.isPresent()) {
```

Comment:
this will include latency for errored requests as well.. which is fine..  We can to know whether or not we are quickly failing